MultiVersions/Forge/main/qualitysdelight/lang/zh_tw.json

```
@@ -0,0 +1,371 @@
+  "item.qualitysdelight.beef_stew_diamond": "燉牛肉",
+  "item.qualitysdelight.beef_stew_gold": "燉牛肉",
+  "item.qualitysdelight.beef_stew_iron": "燉牛肉",
+  "item.qualitysdelight.bone_broth_diamond": "排骨湯",
+  "item.qualitysdelight.bone_broth_gold": "排骨湯",
+  "item.qualitysdelight.bone_broth_iron": "排骨湯",
+  "item.qualitysdelight.bowl_of_guardian_soup_diamond": "碗裝深海守衛濃湯",
+  "item.qualitysdelight.bowl_of_guardian_soup_gold": "碗裝深海守衛濃湯",
+  "item.qualitysdelight.bowl_of_guardian_soup_iron": "碗裝深海守衛濃湯",
+  "item.qualitysdelight.braised_sea_pickle_diamond": "燉海鞘",
+  "item.qualitysdelight.braised_sea_pickle_gold": "燉海鞘",
+  "item.qualitysdelight.braised_sea_pickle_iron": "燉海鞘",
+  "item.qualitysdelight.cabbage_diamond": "高麗菜",
+  "item.qualitysdelight.cabbage_gold": "高麗菜",
+  "item.qualitysdelight.cabbage_iron": "高麗菜",
+  "item.qualitysdelight.cabbage_leaf_diamond": "高麗菜葉",
+  "item.qualitysdelight.cabbage_leaf_gold": "高麗菜葉",
+  "item.qualitysdelight.cabbage_leaf_iron": "高麗菜葉",
+  "item.qualitysdelight.cabbage_rolls_diamond": "高麗菜捲",
+  "item.qualitysdelight.cabbage_rolls_gold": "高麗菜捲",
+  "item.qualitysdelight.cabbage_rolls_iron": "高麗菜捲",
+  "item.qualitysdelight.cabbage_wrapped_elder_guardian_diamond": "遠古深海守衛高麗菜捲",
+  "item.qualitysdelight.cabbage_wrapped_elder_guardian_gold": "遠古深海守衛高麗菜捲",
+  "item.qualitysdelight.cabbage_wrapped_elder_guardian_iron": "遠古深海守衛高麗菜捲",
+  "item.qualitysdelight.chicken_cuts_diamond": "生雞肉塊",
+  "item.qualitysdelight.chicken_cuts_gold": "生雞肉塊",
+  "item.qualitysdelight.chicken_cuts_iron": "生雞肉塊",
+  "item.qualitysdelight.chicken_sandwich_diamond": "雞肉三明治",
+  "item.qualitysdelight.chicken_sandwich_gold": "雞肉三明治",
+  "item.qualitysdelight.chicken_sandwich_iron": "雞肉三明治",
+  "item.qualitysdelight.chicken_soup_diamond": "雞湯",
+  "item.qualitysdelight.chicken_soup_gold": "雞湯",
+  "item.qualitysdelight.chicken_soup_iron": "雞湯",
+  "item.qualitysdelight.cod_roll_diamond": "鱈魚握壽司",
+  "item.qualitysdelight.cod_roll_gold": "鱈魚握壽司",
+  "item.qualitysdelight.cod_roll_iron": "鱈魚握壽司",
+  "item.qualitysdelight.cod_slice_diamond": "生鱈魚片",
+  "item.qualitysdelight.cod_slice_gold": "生鱈魚片",
+  "item.qualitysdelight.cod_slice_iron": "生鱈魚片",
+  "item.qualitysdelight.cooked_bacon_diamond": "烤培根",
+  "item.qualitysdelight.cooked_bacon_gold": "烤培根",
+  "item.qualitysdelight.cooked_bacon_iron": "烤培根",
+  "item.qualitysdelight.cooked_chicken_cuts_diamond": "烤雞肉塊",
+  "item.qualitysdelight.cooked_chicken_cuts_gold": "烤雞肉塊",
+  "item.qualitysdelight.cooked_chicken_cuts_iron": "烤雞肉塊",
+  "item.qualitysdelight.cooked_cod_slice_diamond": "烤鱈魚片",
+  "item.qualitysdelight.cooked_cod_slice_gold": "烤鱈魚片",
+  "item.qualitysdelight.cooked_cod_slice_iron": "烤鱈魚片",
+  "item.qualitysdelight.cooked_elder_guardian_slice_diamond": "烤遠古深海守衛片",
+  "item.qualitysdelight.cooked_elder_guardian_slice_gold": "烤遠古深海守衛片",
+  "item.qualitysdelight.cooked_elder_guardian_slice_iron": "烤遠古深海守衛片",
+  "item.qualitysdelight.cooked_guardian_tail_diamond": "烤深海守衛尾",
```

我在想翻作 `尾鰭` 會不會比較順口，我知道這是該模組的內建翻譯，但就唸起來有點奇怪
```suggestion
  "item.qualitysdelight.cooked_guardian_tail_diamond": "烤深海守衛尾鰭",
```